src/app/dev/DevToys.Blazor/Components/UIElements/UIMultiLineTextInputPresenter.razor.cs

```diff
@@ -178,6 +180,54 @@ private Task OnMonacoEditorInitializedAsync()
         return Task.CompletedTask;
     }
 
+    private async Task OnMonacoEditorPositionChanged(CursorPositionChangedEvent cursorPosition)
+    {
+        if (UIMultiLineTextInput.Tooltips.Count == 0)
+        {
+            return;
+        }
+
+        await _monacoInitializationAwaiter.Task;
+        TextModel model = await _monacoEditor.GetModelAsync();
+        if (model is not null && cursorPosition.Position is not null)
+        {
+            WordAtPosition wordAtPosition = await model.GetWordAtPositionAsync(JSRuntime, cursorPosition.Position);
+            if (wordAtPosition is not null)
+            {
+                UIHoverTooltip? tooltip = UIMultiLineTextInput.Tooltips.FirstOrDefault(tooltip => tooltip.Word.Equals(wordAtPosition.Word, StringComparison.OrdinalIgnoreCase));
+                if (tooltip != null)
+                {
+                    var decoration = new ModelDeltaDecoration
+                    {
+                        Range = new Range
+                        {
+                            StartLineNumber = 1,
+                            StartColumn = 1,
+
+                            EndLineNumber = wordAtPosition.EndColumn,
```

Comment:
This assignment is incorrect. You want to set a line here, not a column.

Reply:
Hum WordAtPosition doesn't expose a LineNumber, that's why I used the EndColumn. Will check if it's possible to have the line number. But we should use another APi from Monaco but it's not exposed for the moment.

Reply:
Yeah. Look at my other comment where I explain how to achieve it.